src/components/OrgFinder/FindOrganization.tsx

```diff
@@ -74,53 +175,207 @@ class FindOrganization extends Component<Props, State> {
+    let lng2 = coordinate2.lng;
+    const avgEarthRadiusInKM = 6371;
+    var radLat = this.degToRad(lat2-lat1);
+    var radLng = this.degToRad(lng2-lng1);
+    var a = 
+      Math.sin(radLat/2) * Math.sin(radLat/2) + 
+      Math.cos(this.degToRad(lat1)) * Math.cos(this.degToRad(lat2)) *
+      Math.sin(radLng/2) * Math.sin(radLng/2)
+      ;
+    var c = 2 * Math.atan2(Math.sqrt(a), Math.sqrt(1-a));
+    var distInKM = avgEarthRadiusInKM * c;
+    return distInKM;
+  }
+
+  // helper function to convert degree to radian
+  degToRad(degree: number ): number {
+    return degree * (Math.PI/180);
+  }
   render() {
     const {
       displayMap,
+      displayError,
+      displayIcon,
       zipcodeSearch,
       organizations,
-      zipcodeLatLng,
+      zipcodeLat,
+      zipcodeLng,
     } = this.state;
 
     return (
       <div className="container">
+
         <div className="jumbotron-fluid mt-5">
-          <h1 className="display-4">Find Nearby Organizations</h1>
+          <h1 className="text-center"><b>Find Organizations Near You</b></h1>
         </div>
         <form onSubmit={this.onSubmitZipcode}>
-          <div className="input-group mb-3 mt-5">
-            <input type="text" className="form-control form-purple" placeholder="Enter your zipcode here" value={zipcodeSearch} onChange={this.onHandleChangeZipcode} />
+          <div className="input-group w-50 mb-3 mt-5 mx-auto">
+            <input
+              type="text"
+              className="form-control form-purple"
+              placeholder="Search by zipcode..."
+              value={zipcodeSearch}
+              onChange={this.onHandleChangeZipcode}
+            />
             <div className="input-group-append">
-              <button className="btn btn-primary btn-primary-theme rounded-0" type="submit">Search</button>
+              <button className="btn btn-info btn-primary-theme" type="submit">Search</button>
             </div>
           </div>
         </form>
+
+        {displayIcon
+          ? (
+            <div className="text-center mb-3 mt-5">
+              <img src={FindOrgIcon} className="img-fluid" alt="Girl with Magnifying Glass" />
+            </div>
+          )
+          : <div />}
+
         {displayMap
           ? (
-            <MapComponent
-              organizations={organizations}
-              lat={zipcodeLatLng.lat}
-              lng={zipcodeLatLng.lng}
-              googleMapURL={`https://maps.googleapis.com/maps/api/js?key=${APIKey}&v=3.exp&libraries=geometry,drawing,places`}
-              loadingElement={<div style={{ height: '100%' }} />}
-              containerElement={<div style={{ height: '400px' }} />}
-              mapElement={<div style={{ height: '100%' }} />}
-            />
+            <div className="row">
+              <div className="col-sm-6">
+                <div className="row">
+                  <h5 className="pb-3 mr-1 ml-3">3</h5>
+                  <h5 className="pb-3">results near 19104</h5>
```

Comment:
can you change these values so that they are not static but they pull from state?